addons/fluxcd/definitions/helm-release-def.yaml

```diff
@@ -137,6 +140,8 @@ spec:
         		// +usage=The bucket region, optional
         		region?: string
         	}
+            // +usage=Alternative list of values files to use as the chart values (values.yamlis not included by default), expected to be a relative path in the SourceRef.Values files are merged in the order of this list with the last file overriding the first.
+            valuesFiles?: [...string]
```

Comment:
If set multiple files, what mean?

Reply:
Advice write the version changes in the readme

Reply:
as the usage tells. The last file will override the first.

Reply:
> Advice write the version changes in the readme

Every changes add a change log will let readme too large that contains too much versioned infos. A readme belong to a special version of addon. IMO no need contain other versions' info.